src/foolscap/slicers/root.py

```diff
@@ -30,6 +40,10 @@ def allowStreaming(self, streamable):
     def registerRefID(self, refid, obj):
         pass
 
+    def next(self):
+        # PY3KPORT
+        return self.__next__()
+    
```

Comment:
See comments higher about "maybe we can refactor a little so we're not recording bound methods into a 3-tuple"...?